package/tests/test_api/test_graphql.py

```diff
@@ -202,6 +248,104 @@ def test_graphql_run_tracking_data_query_show_diff_false(
                     ),
                 )
             ]
+    def test_graphql_run_tracking_data_show_diff_true_one_empty_run(
```

Comment:
You can use https://docs.pytest.org/en/6.2.x/parametrize.html to make these tests cleaner. They are essentially 4 variations of the same test, not 4 separate tests IMHO.

Reply:
Thanks, this looks good. Will parametrize the below!!